crates/rolldown/src/finalizer/impl_visit_mut_for_finalizer.rs

```diff
@@ -188,8 +188,7 @@ impl<'ast, 'me: 'ast> VisitMut<'ast> for Finalizer<'me, 'ast> {
       .ctx
       .linking_info
       .wrapper_stmt_info
-      .map(|idx| self.ctx.module.stmt_infos[idx].is_included)
-      .unwrap_or_default();
+      .is_some_and(|idx| self.ctx.module.stmt_infos[idx].is_included);
```

Comment:
If clippy doesn't throw warnings on this, please keep the code original.

Reply:
warning: build failed, waiting for other jobs to finish...
error: called `map(<f>).unwrap_or_default()` on an `Option` value
   --> crates/rolldown/src/finalizer/impl_visit_mut_for_finalizer.rs:191:8
    |
191 |         .map(|idx| self.ctx.module.stmt_infos[idx].is_included)
    |  ________^
192 | |       .unwrap_or_default();
    | |__________________________^ help: use: `is_some_and(|idx| self.ctx.module.stmt_infos[idx].is_included)`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#manual_is_variant_and
    = note: `-D clippy::manual-is-variant-and` implied by `-D clippy::pedantic`
    = help: to override `-D clippy::pedantic` add `#[allow(clippy::manual_is_variant_and)]`